frontend/src/router/guards.js

```diff
@@ -180,3 +181,9 @@ function ensureKubeconfigDataLoaded (store) {
     return store.dispatch('fetchKubeconfigData')
   }
 }
+
+function ensureControllerRegistrationsLoaded (store) {
+  if (isEmpty(store.getters.controllerRegistrationList)) {
+    return store.dispatch('fetchControllerRegistrations')
```

Comment:
only call `fetchControllerRegistrations` if you have the authorization to do so, similar like
```js
if (store.getters.canUseProjectTerminalShortcuts) {
  promises.push(store.dispatch('ensureProjectTerminalShortcutsLoaded'))
}
```

Reply:
Yes, you are right of course. We need to discuss this PR in general - as we know know that the registrations cannot be accessed by regular users, we need to establish a way for non admins to fetch the network extensions